packages/graphile-build-pg/src/plugins/introspectionQuery.js

```diff
@@ -187,7 +192,11 @@ with
       rel.relpersistence in ('p') and
       -- We don't want classes that will clash with GraphQL (treat them as private)
       rel.relname not like E'\\\\_\\\\_%' and
-      rel.relkind in ('r', 'v', 'm', 'c', 'f') and
+      ${
+        pgUsePartitionParents
+          ? "rel.relkind in ('r', 'v', 'm', 'c', 'f', 'p') and not rel.relispartition"
```

Comment:
Since this is opt in, we don't need to concern ourselves with whether it supports PG9.6 (which it won't). Nice :+1: